### PR TITLE
stopwatch

### PR DESCRIPTION
**Pull request by Shubham-10-pro**
There should be a program for Create Upper case to lower case as there is one for lower case to upper case

### CODE REVIEW

**Comment by dibashthapa** [comment]
@Shubham-10-pro 
1. Improve the filename 
2. add the extension of the filename

**Comment by Shubham-10-pro** [comment]
I think i have correct it now... Nedless to say suggest me to made any further change (if required) @dibashthapa

**Comment by dibashthapa** [comment]
> I think i have correct it now... Nedless to say suggest me to made any further change (if required) @dibashthapa

1. Format the code properly using the formatter
2. Improve the variable name 
3. Too many spacing 
4. Not readable code 


> I think i have correct it now... Nedless to say suggest me to made any further change (if required) @dibashthapa

File: stopwatch.cpp [hunk 1]
```diff
@@ -0,0 +1,104 @@
+#include<conio.h>
```
Comment (thenishantsapkota) [inline stopwatch.cpp:1]: This is not need here.
As modern compliers don't even use that.

File: stopwatch.cpp [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+#include<process.h>
+#include<iostream.h>
```
Comment (thenishantsapkota) [inline stopwatch.cpp:3]: I don't think modern compilers recognize `iostream.h` so change it to just `iostream` as the code will error out when running it on modern compilers.

Comment (Shubham-10-pro) [reply stopwatch.cpp:3]: Is it ok now?

**Pull request closed without merging**